src/_pytest/warning_types.py

```diff
@@ -9,6 +9,15 @@ class PytestWarning(UserWarning):
     """
 
 
+class UnknownMarkWarning(PytestWarning):
```

Comment:
As we discussed in https://github.com/pytest-dev/pytest/pull/4933#discussion_r266096709, let's just raise a `PytestWarning` for now. 👍